### ISSUE TITLE
Add a Collection.xml fixture

### ISSUE THREAD

**Issue by gjnoonan** [julmud/phpDVDProfiler#6]
As mentioned in #5  whilst doing development and testing MRs I have been using a `Collection.xml` file which contains Some Blu-ray's, we should expand on this to include more date, covering what phpDVDProfiler may come across in it's usage.

We should make sure it covers:

- A blank profile with just a title
- DVD
- DVD Boxset
- HD-DVD
- Blu-Ray
- Blu-Ray Boxset
- 4K UHD Blu-Ray
- Non-English versions of the above

Any other suggestions @julmud ?

**Comment by julmud** [comment]
We should also have profiles with 3D sound (Dolby Atmos, DTS-X, or Aura-3D) as it seems it caused problems when importing such profiles. And also some profiles with a rating system with diacritics in the rating system name, and corresponding graphics in gfx/Ratings (I think that currently matches only the Swiss rating system).

No 4K UHD Blu-Ray boxset?

**Comment by gjnoonan** [comment]
4k Boxset, yep we can do. I don't think we currently handle box set's differently depending on their media type though do we?

Sounds profiles are a good shout 👍 

Also:

* Private notes
* watched Statistics
* a title on Loan
* something with a genre of "Adult" (don't necessarily need all the details)

**Comment by julmud** [comment]
> I don't think we currently handle box set's differently depending on their media type though do we?

I don't think we do (and we probably shouldn't anyway). I just mentioned the UHD boxsets because in your first list you had boxsets for both DVD and BluRays, but not UHD. ;-)

For the watched statistics and suppliers, we should have some names with diacritics (more specifically: in the latin1 charset, but outside of the ASCII one) in them. It has caused some problems in the past (impossible to expand the line in the watched statistics, problems during updates for the suppliers).
